api/specs/webserver/v0/components/schemas/auth.yaml

```diff
@@ -0,0 +1,87 @@
+LoginEnveloped:
+  type: object
+  required:
+    - data
+  properties:
+    data:
+      $ref: '#/LoginFormType'
+    error:
+      nullable: true
+      default: null
+
+
+LoginForm:
```

Comment:
Sorry, I did not see your comment. 
Since I discuss with you that schemas and types are not the same thing I decided to remove that suffix. Actually, that test was for some reason disabled .. did u remove it? In any case, we can rename these differently later

Reply:
well, all the tests are disabled... that's the point. I think there were issues with relative paths or so?